tests/utils.py

```diff
@@ -10,10 +10,10 @@
 
 
 def get_root_dir():
-    import conda_build
+    import conda_index
 
-    conda_build_dir = os.path.realpath(os.path.dirname(conda_build.__file__))
-    return os.path.abspath(os.path.join(conda_build_dir, ".."))
+    conda_index_dir = os.path.realpath(os.path.dirname(conda_index.__file__))
```

Comment:
```suggestion
    conda_index_dir = importlib.util.find_spec("conda_index").origin
```

Unless you're depending on side effects of actually importing the module - this might be a cleaner way of resolving the module to it's root dir.

Reply:
I think I will keep that one as-is, we do have to import conda_index for the tests.